parl/layers/layer_wrappers.py

```diff
@@ -20,6 +20,7 @@
 import paddle.fluid.unique_name as unique_name
 import paddle.fluid as fluid
 import six
+from .utils import AttrHolder
```

Comment:
To keep line consistent with other codes that describes the code path, use absolute path here, e.g., parl.layers.utils

Reply:
done